src/content/en/fundamentals/accessibility/a11y-for-teams.md

```diff
@@ -0,0 +1,341 @@
+![A design comp with controls marked with their accessible labels.](imgs/labels.jpg)
+
+You might consider creating a mock with all of your controls labeled. This can
+be shared with your development team and QA team for implementation and testing.
+
+
+### Multiple ways to interact with and understand UI
+
+It's easy to assume that all users interact with the
+page primarily using a mouse. When designing, consider how someone will
+interact with a control using a keyboard instead.
+
+Plan your focus states! This means determining what a control will look like
+when the user focuses it with tab or presses the arrow keys. It's useful
+to have these states planned out early, rather than trying to shoehorn them
+into the design later.
+
+Finally, for any point of interaction, you want to make sure that the user has
+multiple ways of understanding the content. Try not to use color alone to convey
+information, as these subtle cues may be missed by a user with a color vision
+deficiency. A classic example is an invalid text field. Instead of just a red
+underline to signify a problem, also consider adding some helper text. That
+way you're covering more bases and increasing the likelihood that a user will
+notice the issue.
+
+## Developer
+
+The developer's role is where focus management and semantics combine to form a
+robust user experience. Below are a few items a developer can keep in mind as
+they're working on their site or application:
+
+- The tab order is logical.
+- Focus is properly managed and visible.
+- Interactive elements have keyboard support.
+- ARIA roles & attributes are applied as needed.
+- Elements are properly labeled.
+- Testing is automated.
+
+### Logical tab order
+
+Native elements like `input`, `button`, and `select` get opted into the tab
+order for free and are automatically focusable with the keyboard. But not all
+elements receive this same behavior! In particular, generic elements like `div`,
+and `span`, are not opted into the tab order. This means if you use a `div` to
+create an interactive control, you'll need to do some additional work to make
+it keyboard accessible.
+
+Two options are:
+
+- Give the control a `tabindex="0"`. This will at least make it focusable,
+  though you'll likely need to do additional work to add support for keypresses.
+- Where possible, consider using a `button` instead of a `div` or `span` for any
+button-like control. The native `button` element is very easy to style and gets
+keyboard support for free.
+
+<div class="video-wrapper-full-width">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="CZGqnp06DnI"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+### Managing focus
+
+When you change the content of the page, it's important to direct the user's
+attention by moving focus. A classic example of when this technique is useful is
+opening a modal dialog. If a user relying on a keyboard presses a button to open
```

Comment:
Change 'A classic example of when this technique is useful is opening a modal dialog' to 'A classic example of when this technique is useful is **when** opening a modal dialog **box**.'